boards.txt

```diff
@@ -1892,3 +1929,67 @@ coredev.menu.DebugLevel.OTA2____=OTA + Updater
 coredev.menu.DebugLevel.OTA2____.build.debug_level=-DDEBUG_ESP_OTA -DDEBUG_ESP_UPDATER
 coredev.menu.DebugLevel.all_____=All
 coredev.menu.DebugLevel.all_____.build.debug_level=-DDEBUG_ESP_CORE -DDEBUG_ESP_SSL -DDEBUG_ESP_WIFI -DDEBUG_ESP_HTTP_CLIENT -DDEBUG_ESP_HTTP_UPDATE -DDEBUG_ESP_HTTP_SERVER -DDEBUG_ESP_UPDATER -DDEBUG_ESP_OTA -DDEBUG_TLS_MEM
+
+############ Arduino boards with Esp8266 ############
+
+arduino-esp8266.name=Arduino
+
+arduino-esp8266.upload.tool=esptool
+arduino-esp8266.upload.speed=9600
```

Comment:
yes, because into `uno wifi dev. ed.` the maximum speed between `ATmega328P` and `esp8266` is 9600, for the other boards the speed can be higher.